BicycleParkingStation/examples/example-normalized.jsonld

```diff
@@ -29,7 +29,7 @@
         ]
     },
     "@context": [
-        "https://raw.githubusercontent.com/smart-data-models/dataModel.OSLO/master/context.jsonld",
+        "https://raw.githubusercontent.com/smart-data-models/dataModel.OSLO/master/notes_context.jsonld",
```

Comment:
this is not right, the notes_context.jsonld is merged with the automatic @context to customize with the mentions to the wished ontologies.

Reply:
Hi Alberto, the URI's in the automatic context are all new smartdatamodel terms. I want to reuse terms from external vocabularies like in the notes_context.jsonld. What can I do to change the automatic context?